src/backend/api/reviews.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+//Returns all reviews
+router.get("/", async (request, response) => {
+  const allReviews = await knex("review").select("*");
+  response.json(allReviews);
+});
+
+//Adds a new review
+router.post("/", async (request, response) => {
+  try {
+    const newReview = await knex("review").insert(request.body);
```

Comment:
Putting things into your database that came from a user without checking it is a bad idea! 
Perhaps you can add some kind of validation to the data inside of body before putting it into the database?